plugin/web/extensions/sidebar/modals/welcomescreen/hooks.ts

```diff
@@ -0,0 +1,31 @@
+import { postMsg } from "@web/extensions/sidebar/utils";
+import { useCallback, useState } from "react";
+
+export default () => {
+  const [ShowVideo, setShowVideo] = useState(false);
+  const [dontShowAgain, setDontShowAgain] = useState(false);
+
+  const handleDontShowAgain = useCallback(() => {
+    setDontShowAgain(!dontShowAgain);
+    postMsg({
+      action: "storageSaveWelcomeScreen",
+      payload: { key: "doNotShowWelcome", value: dontShowAgain },
+    });
```

Comment:
I feel like this postMsg should happen in the handleClose, and be called ONLY IF dontShowAgain is true. We don't need to save a falsey value.

Reply:
da1bbcf40d847aea1dc797d4d6b3493eaf5889a3